src/libNode/Node.cpp

```diff
@@ -282,13 +282,11 @@ bool Node::StartRetrieveHistory(bool& wakeupForUpgrade) {
   }
 
   /// Retrieve DS blocks
-  bool ds_result;
-  m_retriever->RetrieveBlockLink(ds_result, wakeupForUpgrade);
+  bool ds_result = m_retriever->RetrieveBlockLink(wakeupForUpgrade);
```

Comment:
@KaustubhShamshery Here the comments write to retrieve DS blocks, but function call is to retrieve block link, is there any problem here?

Reply:
I think the comment needs to be changed, can you incorporate this in your PR?

Reply:
Ok, can

Reply:
Thanks !!